wfe/wfe.go

```diff
@@ -623,12 +628,22 @@ func (wfe *WebFrontEndImpl) UpdateAccount(
 		ID:  existingAcct.ID,
 	}
 
-	newAcct.Contact = updateAcctReq.Contact
-	// Verify that the contact information provided is supported & valid
-	prob = wfe.verifyContacts(newAcct.Account)
-	if prob != nil {
-		wfe.sendError(prob, response)
+	switch {
+	case updateAcctReq.Status == deactivatedStatus:
+		newAcct.Status = updateAcctReq.Status
+	case updateAcctReq.Status != newAcct.Status:
```

Comment:
I think this switch case might need to be `updatedAcctReq.Status != "" && updatedAcctReq.Status != newAcct.Status`. 

It looks like as-written this breaks an account update that only changes the contact field, not sending any new status field:

```
$ pebble-client -email foo@bar.com
welcome to the pebble shell
Requesting directory from "https://localhost:14000/dir"
Requesting nonce from "https://localhost:14000/nonce-plz"
Registering new account with "https://localhost:14000/sign-me-up"
Requesting nonce from "https://localhost:14000/nonce-plz"
Your account ID is "https://localhost:14000/my-account/4e74148cfb32e323d86667990bc9b9ce5a28b47cfa6b86f76c09a762cec9f523"
Starting REPL environment...
$> Enter a directory endpoint or a URL to POST: https://localhost:14000/my-account/4e74148cfb32e323d86667990bc9b9ce5a28b47cfa6b86f76c09a762cec9f523
$> Enter JSON body, empty line to finish : 
{ "contact":["mailto:foo@foo.com"] }

Requesting nonce from "https://localhost:14000/nonce-plz"
REPL error: Response 400: {
   "type": "urn:ietf:params:acme:error:malformedRequest",
   "detail": "Invalid account status: ",
   "status": 400
}
```